shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -14,6 +14,18 @@ const SHOOV_CI_BUILD_CI_BUILD_STATUS_ERROR = 2;
 const SHOOV_CI_BUILD_CI_BUILD_STATUS_UNCONFIRMED_ERROR = 3;
 
 /**
+ * Implement hook_menu().
+ */
+function shoov_ci_build_menu() {
+  $items["/ci_build/%mid/trigger"] = array(
```

Comment:
no `%mid`. it either `%` or better `%message`

Also this link should work with CTools ajax. @RoySegall can help here :)